### PR TITLE
 LLS: Some edits for the LLS GUI

### PR DESCRIPTION
**Pull request by profxj**
A new method or two
and a test

### CODE REVIEW

File: linetools/isgm/abssystem.py [hunk 1]
```diff
@@ -172,10 +172,31 @@ def chk_component(self, component):
...
+        # Return
+        return abslist
+
+
 class GenericAbsSystem(AbsSystem):
     """Class for Generic Absorption Line System
     """
     def __init__(self, radec, zabs, vlim, **kwargs):
+        if vlim is None:
+            if vlim is None:
```
Comment (ntejos) [inline linetools/isgm/abssystem.py]: Duplicate statements.

**Comment by profxj** [comment]
Fixed.

Anything else?

File: linetools/isgm/abssystem.py [hunk 2]
```diff
@@ -172,10 +172,30 @@ def chk_component(self, component):
...
+        # Return
+        return abslist
+
+
 class GenericAbsSystem(AbsSystem):
     """Class for Generic Absorption Line System
     """
     def __init__(self, radec, zabs, vlim, **kwargs):
+        if vlim is None:
+            vlim = [-500.,500.]*u.km/u.s
```
Comment (ntejos) [inline linetools/isgm/abssystem.py]: I'm still thinking whether this is a good idea or not. If you do something like this for vlim, then for the sake of symmetry I would also allow radec and zabs to have default values, which may be useful in some cases where you do not care about those quantities. On the other hand, for the cases you do care, having default values may cause some trouble...

Comment (ntejos) [inline linetools/isgm/abssystem.py]: How about having them as optional parameters?

File: linetools/isgm/lls.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 from linetools.analysis import absline as ltaa
```
Comment (ntejos) [inline linetools/isgm/lls.py]: Should you move lls.py to the IGM package instead?

File: linetools/isgm/abssystem.py (hunk 2 above)
Comment (profxj) [inline linetools/isgm/abssystem.py]: Yeah, fair point.

I'll go back to what we had.

File: linetools/isgm/lls.py (hunk 3 above)
Comment (profxj) [inline linetools/isgm/lls.py]: Not today and this needs to be done now.

**Pull request merged**